rg_instructor_analytics/static/rg_instructor_analytics/js/instructor_analytics.js

```diff
@@ -4,21 +4,184 @@
 
     var tabList = [];
 
+
     function Tab(button, content) {
         this.button = button;
         this.content = content;
         this.setActive = function (isActive) {
             if (isActive) {
                 content.addClass('active-section');
                 button.addClass('active-section');
+                this.loadTabData()
             } else {
                 content.removeClass('active-section');
                 button.removeClass('active-section');
             }
+        };
+
+        this.loadTabData = function () {
```

Comment:
I cannot find where does this error is handled, so the question why don't just log some message in the console?

Reply:
It is the abstract method. We do not need to handle this exception.